### DISCUSSION TITLE
WDS - Wireless Distribution System

### DISCUSSION THREAD

**Discussion by mlgwilpernig** [RaspAP/raspap-webgui#1345] [Ideas]
Team,
what will it take to connect several RASPI's with RASPAP installed to a WDS - Wireless Distribution System where one can roam from floor to floor, or even over to the barn.
I have Ethernet and APs in the different locations, but moving between the locations, requires frequent reconnects as the phone, table or widget does not see the other AP as a partner in the system.
Also great would be the possibility to control several RASPAPs centrally as a single networked setup.

Any idea ?

**Comment by billz** [comment]
AFAIK this won't work with the RPi, at least not with the built-in NIC. You might have luck with USB adapters that support WDS, but my guess is this would be [vendor specific](https://oldwiki.archive.openwrt.org/doc/howto/clientmode).
